components/form/Select.js

```diff
@@ -10,7 +10,7 @@ export default function Select({
   return (
     <>
       {label && (
-        <label htmlFor={name} className={`sr-only ${className} hidden md:block`}>
+        <label htmlFor={name} className={`${className} hidden md:block`}>
```

Comment:
```suggestion
        <label htmlFor={name} className={`sr-only ${className} hidden md:block`}>
```
sr-only is used for accessibility purposes I think this shouldnot be removed.